samples/touchtest/touchtest.cpp

```diff
@@ -0,0 +1,209 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        touchtest.cpp
+// Purpose:     Multitouch sample
+// Author:      Martin Koegler
+// Modified by:
+// Created:     2015-07-29
+// Copyright:   (c) Martin Koegler
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#include "wx/wx.h"
+
+// Define a new application
+class MyApp: public wxApp
+{
+public:
+    bool OnInit();
+};
+
+DECLARE_APP(MyApp)
+
+#define TOUCH_POINTS 16
```

Comment:
Could we please use `const unsigned TOUCH_POINTS = 16;` here instead?

Reply:
Only constexpr would fit - see m_TouchPoints array